### ISSUE TITLE
Is continous batching supported?

### ISSUE THREAD

**Issue by sirmo** [LostRuins/koboldcpp#798]
I am not able to find much on batching support. But it appears that the downstream llama.cpp supports it.

https://github.com/ggerganov/llama.cpp/issues/4372

Any plans to expose this feature in koboldcpp? 

I really like using koboldcpp and for speeding up scripting it would be quite nice.

**Comment by LostRuins** [comment]
No, KoboldCpp does not support batching. However, it does support queueing, which is automatically enabled when launching with the GUI launcher. For terminal users, adding `--multiuser` will allow this feature. You can then send multiple parallel requests, and they will be queued and completed as needed.

**Comment by phazei** [comment]
@LostRuins 
What about now?  I saw "cont-batching" in the code, but didn't see it in the docs.

**Comment by LostRuins** [comment]
Batching is not supported in KoboldCpp currently.
